Mathlib/CategoryTheory/Preadditive/AdditiveFunctor.lean

```diff
@@ -162,6 +162,22 @@ instance (priority := 100) preservesFiniteBiproductsOfAdditive [Additive F] :
             simp_rw [F.mapBicone_π, F.mapBicone_ι, ← F.map_comp]
             erw [← F.map_sum, ← F.map_id, IsBilimit.total hb])⟩ } }
 
+instance (priority := 100) preservesFiniteBiproductsOfShapeOfAdditive [Additive F]
```

Comment:
You should use `PreservesFiniteBiproducts`, `PreservesFiniteProducts` and `PreservesFiniteCoproducts`.